app/router.js

```diff
@@ -65,6 +65,8 @@ router.map(function() {
     this.route('saved');
     this.route('past');
   });
-});
```

Comment:
One more thing in the same file router.js travis is showing the error 
`export default router;` should be placed at the top! Shall I place it below import?  
@niranjan94